doc/control-tower.md

```diff
@@ -0,0 +1,160 @@
+# Path to AWS Control Tower
+
+[AWS Control Tower](https://aws.amazon.com/controltower/) provides an easy way to setup a Landing Zone in AWS: it will create a multi-account structure with proper set of permissions (using [AWS SSO](https://aws.amazon.com/single-sign-on/)) and guardrails (using [Service Control Policies](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) (SCPs) and [AWS Config](https://aws.amazon.com/config/)).
+
+Control Tower creates multiples AWS resources for you:
+- One Organization
+- One or two Organizational Units (OUs): Security (mandatory) and Sandbox (optional)
+- Two AWS Accounts in the Security OU: Log Archive and Audit
+- A directory in AWS SSO
+- A set of SCPs
+- ...
+You can get more details on Control Tower [here](https://docs.aws.amazon.com/controltower/latest/userguide/what-is-control-tower.html).
+
+If you need to setup these resources in your AWS environment and get a managed AWS service to govern your AWS Organization and account structure, Control Tower is the recommended solution.
+
+If you already bootstraped your landing zone using the bootstrap kit, this document will guide you through the "migration" to Control Tower.
+
+At the end of this guide, you will have what bootstrap kit created: the Organization, OUs (Shared, SDLC, Prod), accounts (CICD, Dev, Staging, Prod, ...) plus what Control Tower will create (see above).
+
+## Prerequisites
+
+**Disclaimer**: This page is only needed when you already have used bootstrap kit to setup your environment. If you start from a blank page or an existing Organization not created by bootstrap kit, refer to the Control Tower documentation.
+
+When you setup a landing zone with Control Tower, it performs some pre-check (see [doc](https://docs.aws.amazon.com/controltower/latest/userguide/getting-started-with-control-tower.html)):
+
+### CloudTrail integration
+
+One of them is about AWS CloudTrail. CloudTrail must not be enabled for your organization. Otherwise you will get the fllowing warning:
```

Comment:
```suggestion
One of them is about AWS CloudTrail which is enabled by the Bootstrap Kit. CloudTrail must not be enabled for your organization. Otherwise you will get the following warning:
```

Reply:
OK